### PR TITLE
Use stackhpc monasca-grafana v4.2.0 rebase (test)

### PR DESCRIPTION
**Pull request by Wasaac**
This includes the config needed to build and deploy an updated monasca-grafana (v4.2.0).

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -167,6 +167,10 @@ kolla_build_customizations:
   sahara_base_packages_append:
     # Allow custom packages to be installed via pip.
     - 'python-pip'
+  monasca_grafana_version_override:
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml]: I don't think we need this

**Review by dougszumski — CHANGES_REQUESTED** [review]

File: etc/kayobe/kolla.yml (hunk 1 above)
Comment (dougszumski) [reply etc/kayobe/kolla.yml]: Looks good otherwise

Comment (Wasaac) [reply etc/kayobe/kolla.yml]: Updated

**Review by dougszumski — APPROVED** [review]
Good to go - let's test it builds with the correct version before merging.

**Pull request closed without merging**